x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/Connector.java

```diff
@@ -469,6 +479,7 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeEnum(status);
         out.writeGenericValue(syncCursor);
         out.writeBoolean(syncNow);
+        out.writeBoolean(isDeleted);
```

Comment:
Curious: do we need a new transport version, if we add something at the end of an entity, which is serialized over the wire? Does a potentially old node simply discard everything after `syncNow`? Never tested that (I assume it works without a transport version check, but just to double-check)

Reply:
Should `ConnectorTests` extend `AbstractWireSerializingTestCase` btw as it can be serialized over the wire?